pkgs/development/compilers/souffle/default.nix

```diff
@@ -1,40 +1,43 @@
 { lib, stdenv, fetchFromGitHub
-, perl, ncurses, zlib, sqlite, libffi
-, autoreconfHook, mcpp, bison, flex, doxygen, graphviz
-, makeWrapper
+, bash-completion, perl, ncurses, zlib, sqlite, libffi
+, mcpp, cmake, bison, flex, doxygen, graphviz
+, makeWrapper, writeShellScript
 }:
 
 
 let
   toolsPath = lib.makeBinPath [ mcpp ];
+  # The build system of Soufflé 2.1 requires git on the $PATH, but will without
+  # it as long as it exists with a non-zero code. This should be unnecessary in
+  # the next release, see SOUFFLE_GIT in CMakeLists.txt.
+  git = writeShellScript "git" "exit 1";
```

Comment:
Can we patch this instead?

Reply:
Sure, happy to. Why is patching preferred, out of curiosity?

Reply:
Creating fake binaries is more a hack. Patching the right version number in the build files is nicer and makes sure it is set correct.